src/test/java/teammates/common/datatransfer/questions/FeedbackRankQuestionDetailsTest.java

```diff
@@ -0,0 +1,44 @@
+package teammates.common.datatransfer.questions;
+
+import org.testng.annotations.Test;
+
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@link FeedbackRankQuestionDetails}.
+ */
+public class FeedbackRankQuestionDetailsTest extends BaseTestCase {
+    @Test
+    public void testValidateMinOptionsToBeRanked_sameValidValues_shouldReturnTrue() {
+        int testValue = 100;
+        FeedbackRankQuestionDetails feedbackRankQuestionDetails = new FeedbackRankOptionsQuestionDetails();
+        feedbackRankQuestionDetails.setMinOptionsToBeRanked(testValue);
+        assertEquals(feedbackRankQuestionDetails.minOptionsToBeRanked, testValue);
+    }
+
+    @Test
+    public void testValidateMinOptionsToBeRanked_differentValidValues_shouldReturnTrue() {
```

Comment:
This test is not really doing anything right? We already test that a valid value is set in the above test. (same for the other one below)